eZ/Publish/Core/Pagination/Pagerfanta/ContentSearchHitAdapter.php

```diff
@@ -24,13 +24,30 @@ class ContentSearchHitAdapter implements AdapterInterface
     /** @var \eZ\Publish\API\Repository\SearchService */
     private $searchService;
 
+    /** @var array */
+    private $languageFilter;
+
+    /** @var string|null */
```

Comment:
would need some doc on this +  on `getDisplayLanguage()`, `$languageFilter` is okish to skip it on as it's kind of a known thing elsewhere in kernel (API), but within context of kernel, what is `$displayLanguage` ?

What's even a display? ;) :P

Reply:
@michal-myszka `$displayLanguage` is `ezplatform-admin-ui` specific and shouldn't be placed here. Please move it to `\EzSystems\EzPlatformAdminUi\Tab\Dashboard\PagerContentToDataMapper`.

Reply:
@andrerom you're right - I've changed the "displayName" name, you will see that in related admin-ui PR
@adamwojs also right :) moved